autorag/evaluation/metric/generation.py

```diff
@@ -64,12 +64,25 @@ def compute_score(gt: List[str], pred: str) -> float:
     return result
 
 
-@generation_metric
-def bleu(gt: List[str], pred: str, **kwargs) -> float:
+#@generation_metric
+def bleu(generation_gt: List[List[str]], generations: [str], tokenize: str|None = None, smooth_method: str = 'exp', smooth_value: Optional[float] = None, max_ngram_order: int = 4, trg_lang: str = '', **kwargs) -> List[float]:
     """
-    Compute bleu score for generation.
+    Computes the BLEU metric given pred and ground-truth.
+
+    :param tokenize: The tokenizer to use. If None, defaults to language-specific tokenizers with '13a' as the fallback default. check #https://github.com/mjpost/sacrebleu/blob/master/sacrebleu/metrics/bleu.py
```

Comment:
Please add annotation for generation_gt and generations.
```
:param generation_gt: A list of ground truth.
        Must be 2-d list of string.
        Because it can be a multiple ground truth.
:param generations: A list of generations that LLM generated.
```